src/Form/ApplicationRequest/ApplicationRequestType.php

```diff
@@ -94,5 +95,10 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 $data->setCityName((string) FranceCitiesBundle::getCity($postalCode, $inseeCode));
             }
         });
+
+        $builder
+            ->get('favoriteCities')
+            ->addModelTransformer(new StringToArrayTransformer())
```

Comment:
this transformer is not needed for me here.

`favoriteCities` is already a CollectionType of TextType entries.

so basically it already transform this into an array of strings.

Then doctrine handles it to transform & store it into the database, but its not our problem!